test/github-auth.js

```diff
@@ -100,6 +105,11 @@ class FakeGithub {
   setUser({id, email}) {
     this.github_users.push({id: id.toString(), email});
   }
+
+  setRepositories(repoName) {
+    let repos = [...repoName].map(repo => {name: repo});
```

Comment:
I think `repoName` is actually an array of names, right?
```
> repoName = "awesomeRepo";
'awesomeRepo'
> [...repoName]
[ 'a', 'w', 'e', 's', 'o', 'm', 'e', 'R', 'e', 'p', 'o' ]
```
maybe just change the name of the variable so the reader doesn't get confused?